1-data-marketplace/infra/analytics/co-managed/main.bicep

```diff
@@ -186,3 +186,4 @@ output serviceProviderDatalakeName string = serviveProviderDatalake.outputs.name
 output serviceProviderDatalakeResourceId string = serviveProviderDatalake.outputs.id
 output resourceGroupName string = useExistingCoManagedResourceGroup ? analyticsExistingCoManagedRg.name : analyticsCoManagedRg.name
 output keyVaultName string = keyvault.outputs.keyVaultName
+output purviewResourceName string = purview.outputs.purviewName
```

Comment:
it might fail here in case useExistingPurview is true
maybe this should be 
`output purviewResourceName string = useExistingPurview ? purview.outputs.purviewName : null` or something like this and then in the caller take care of the null